franklab_mountainsort/ms4_franklab_proc2py.py

```diff
@@ -324,11 +324,9 @@ def tagged_curation(cluster_metrics, metrics_tagged,
     noise_overlap_thresh : float, optional
     peak_snr_thresh : float, optional
     mv2file : str, optional
-    opts : None or dict, optional
```

Comment:
Why was opts removed?

Reply:
Because this opts is actually not used in the function.